### PR TITLE
Create cord-19.yaml

### PR DESCRIPTION
**Pull request by skohlmeier**
Note: Our S3 bucket is available here: https://s3.console.aws.amazon.com/s3/buckets/ai2-semanticscholar-cord-19/?region=us-west-2&tab=overview.

It's unclear how I should link to specific resources given that we will be creating new folders as we version/update the dataset.

### CODE REVIEW

File: datasets/cord-19.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ Tags:
 License: Open (see license file for details)
 Resources:
   - Description: S3 bucket with CORD-19 dataset files
-    ARN: https://s3.console.aws.amazon.com/s3/buckets/ai2-semanticscholar-cord-19/
```
Comment (chueatwork) [inline datasets/cord-19.yaml]: arn:aws:s3:::ai2-semanticscholar-cord-19

File: datasets/cord-19.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - Description: S3 bucket with CORD-19 dataset files
+    ARN: ai2-semanticscholar-cord-19
+    Region: us-west-2
+    Type: S3
+DataAtWork:
+  Tools & Applications:
+    - Title: COVID-19 Open Research Dataset Challenge (CORD-19)
+      URL: https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge
+      AuthorName: Kaggle
+      AuthorURL: N/A
```
Comment (chueatwork) [inline datasets/cord-19.yaml]: AuthorURL is optional, just do not include if no relevant URL

File: datasets/cord-19.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@ Tags:
 License: Open (see license file for details)
 Resources:
   - Description: S3 bucket with CORD-19 dataset files
-    ARN: https://s3.console.aws.amazon.com/s3/buckets/ai2-semanticscholar-cord-19/
+    ARN: ai2-semanticscholar-cord-19
     Region: us-west-2
     Type: S3
```
Comment (chueatwork) [inline datasets/cord-19.yaml]: Resource type can be be CloudFront Distribution, DB Snapshot, S3 Bucket, or SNS Topic; just revise "S3" to "S3 Bucket"!

File: datasets/cord-19.yaml (hunk 1 above)
Comment (skohlmeier) [reply datasets/cord-19.yaml]: Done!

File: datasets/cord-19.yaml (hunk 2 above)
Comment (skohlmeier) [reply datasets/cord-19.yaml]: Done!

File: datasets/cord-19.yaml (hunk 3 above)
Comment (skohlmeier) [reply datasets/cord-19.yaml]: Done!

**Pull request merged**